Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp

```diff
@@ -1673,7 +1673,7 @@ ExceptionOr<void> CanvasRenderingContext2DBase::drawImage(Document& document, Ca
     } else
         c->drawImage(*image, normalizedDstRect, normalizedSrcRect, options);
 
-    didDraw(repaintEntireCanvas, normalizedDstRect);
+    didDraw(repaintEntireCanvas, normalizedDstRect, defaultDidDrawOptionsWithoutPostProcessing());
```

Comment:
The image itself might be from SVG or from image buffer or such, which again leaves the rasterizer fingerprint to the bitmap and as such should have the postprocessing

Reply:
Thanks for raising this issue. I added a check for whether or not the image was a bitmap image.